t/24-worker-jobs.t

```diff
@@ -1253,4 +1255,51 @@ subtest 'Cache setup error handling' => sub {
       'show the error message that is returned by do_asset_caching correctly';
 };
 
+subtest 'symlink DIR to workers working directory' => sub {
```

Comment:
Likely `t/24-worker-engine.t` would have been more fitting since you're actually modifying `isotovideo.pm` and not `Job.pm`. You likely also don't need to deal with the "life-cycle" of a job object here¹.

¹ I'm referring to code like

```
    $job->accept;
    wait_until_job_status_ok($job, 'accepted');
    combined_like { $job->start 
```